prototypes/recipe/bulk/bulk.lua

```diff
@@ -1450,6 +1450,17 @@ data:extend(
 	ingredients = {{"coke", 4}},
 	result = "graphite-block"
 },
+--Graphite Grinding
```

Comment:
Good idea! Then you only need to ship graphite blocks, which are more dense and grind it to powder on-side

Reply:
More importantly, the basic recipes gives 8x coke -> 4x powder, and then 4x powder -> 1x block. The electric furnace recipe in Carbon Processing gives 4x coke -> 1x block, so adding this transformation saves 50% of your input coke.